.gitignore

```diff
@@ -11,3 +11,5 @@ CMakeCache.txt
 *.cbp
 CMakeFiles/
 *~
+TAGS
+*#
```

Comment:
what the use of TAGS and also add a new line

Reply:
TAGS is generated by emacs to help navigate source
will add newline
for ref: 
https://github.com/jixiuf/ctags-update